File: Java/rev_num.java
import java.util.*;
public class rev_num{
    public static void main(String[] args) {
        try(Scanner sc=new Scanner(System.in)){
            System.out.print("Enter the digit: ");
            int a=sc.nextInt(),r=0;
            while(a>0){
                r=r*10+a%10;
                a/=10;
            }
            sc.close();
            System.out.print("Reverse of the number is : "+r);
        }
    }
}

File: C/rev_num.c
#include<stdio.h>
int main(){
    long long int a,r=0;
    printf("Enter the digit: ");
    scanf("%lld",&a);
    while(a>0){
        r=(r*10)+a%10;
        a/=10;
    }
    printf("Reverse of the digit is: %lld",r);
    return 0;
}